README.md

```diff
@@ -19,9 +19,10 @@
 #
 -->
 
-[![Build Status](https://travis-ci.org/apache/cordova-node-xcode.svg?branch=master)](https://travis-ci.org/apache/cordova-node-xcode)
+# Cordova node xcode tool
```

Comment:
```suggestion
# node xcode (by Apache Cordova)
```

Reply:
I actually unapplied this suggestion because it was *originally* made by someone else (<https://github.com/alunny/node-xcode>).

I would also like to have the top-level title as consistent with the package name as we can. I changed `tool` to `utility`, would like to keep something to make it clear that this project is not Xcode itself.